frost-ristretto255/src/frost.rs

```diff
@@ -655,91 +918,25 @@ pub fn aggregate(
     );
 
     for signing_share in signing_shares {
-        let signer_pubkey = pubkeys.signer_pubkeys[&signing_share.index];
+        let signer_pubkey = pubkeys.signer_pubkeys.get(&signing_share.index).unwrap();
         let lambda_i = generate_lagrange_coeff(signing_share.index, signing_package)?;
-        let signer_commitment = signing_package
-            .signing_commitments
-            .iter()
-            .find(|comm| comm.index == signing_share.index)
-            .ok_or("No matching signing commitment for signer")?;
-
-        let commitment_i = signer_commitment.hiding.0
-            + (signer_commitment.binding.0 * bindings[&signing_share.index]);
 
-        signing_share.check_is_valid(&signer_pubkey, lambda_i, commitment_i, challenge)?;
+        signing_share.verify(signer_pubkey, lambda_i, challenge)?;
```

Comment:
This is incomptable with `verify_signature_share` as defined in the spec.

Reply:
Because the group commitment is not passed in/calc'd from Round 1 commitments? I'll fix

Reply:
Updated